esp/src/eclwatch/XrefFoundFilesWidget.js

```diff
@@ -121,7 +121,7 @@ define([
             var selections = this.grid.getSelected();
             var list = this.arrayToList(selections, "Name");
             if (confirm(this.i18n.AddTheseFilesToDali + "\n" + list)) {
-                WsDFUXref.DFUXRefArrayAction(selections, this.i18n.Attach, context.params.Name, "Attach").then(function (response) {
+                WsDFUXref.DFUXRefArrayAction(selections, this.i18n.Attach, context.params.Name, "Found").then(function (response) {
```

Comment:
So `this.i18n.Attach` is a call to the translation "Attach" we need to bypass the translation (which was added on accident) and use the string "Attach" instead of it being a translation. Again, something you did not introduce but it will fully fix this line.

`WsDFUXref.DFUXRefArrayAction(selections, "Attach", context.params.Name, "Found") should `
should do it.